### ISSUE TITLE
Tela de Detalhes do Medo

### ISSUE THREAD

**Issue by lucasmontano** [lucasmontano/first-how-to-build-an-app#3]
Como usuário deste aplicativo, o passo final do cadastro é definir o que é este medo e como ele está bloqueando um objetivo.

## Fluxo de navegação
Esta tela é acessível através da tela de Classificação do Medo ou através do botão de editar um medo.

## Definição da tela
Para contextualizar o usuário a classificação selecionada na etapa anterior deve ser exibida no topo da tela (não será possível editar a classificação nesta versão). 

### Perguntas que o usuário deve responder

**1 - Este medo ta relacionado a qual objetivo?**
A resposta deste campo deve ser curta, e ela vai ser utilizada em outras telas, para manter uma comunicação positiva.

Um texto de apoio para ajudar o usuário a preencher este campo de texto deve ser exibido.

_Exemplo de resposta: conseguir um aumento salarial._

**2 - Liste as piores coisa que podem acontecer enquanto você tenta alcançar esse objetivo?**
O usuário pode preencher no mínimo um item e no máximo cinco. O primeiro estado deste componente é um input de texto vazio, quando o usuário começar a digitar um texto, um novo input vazio deve aparecer abaixo. 

_Exemplo de resposta:_
- _ser demitido porque pedi um aumento_
- _ter aumento negado e começar a perder status com meu time e chefe_
- _[próximo campo sempre vazio para uma nova resposta]_

### Ação principal da tela
- Salvar

### Ação secundária 
- Voltar

## Wireframe
![Untitled design-4](https://user-images.githubusercontent.com/7559318/55292454-85e04c00-53eb-11e9-9f0a-3180de8d7ee2.png)

**Comment by clautercarlos** [comment]
@lucasmontano uma sugestão(muito pessoal) para um futuro release: o usuário poderia criar um tipo de diário relacionado a cada medo cadastrado com base no conteúdo da curadoria.    
   
**Exemplo:**   
* Morro de medo de fazer uma entrevista de emprego em inglês.  
    - Recebo um material a respeito   
    - Faço uma anotação sobre como o material poderia me ajudar e o que posso fazer com essa ajuda no meu "log do medo".    
    - A cada 3 dias eu poderia receber um lembrete dessa anotação pra saber se eu fiz algo para diminuir esse medo.

É muito pessoal porque eu gosto de ter acompanhamento de algumas coisas.

**Comment by lucasmontano** [comment]
@clautercarlos concordo! É importante para a retenção do usuário ter esse sentimento de progresso, e  boa parte desse sentimento é dar a oportunidade para ele "marcar" algo como feito, e visualizar tudo que já concluiu assim como o que tem ainda para fazer. 

Vamos colocar isto para a versão **v1.1.0**

**Comment by clautercarlos** [comment]
Tu fez uma excelente tradução do que eu tentei dizer. hahaha

**Comment by devluizsilva** [comment]
cara muito massa o projeto já tem mais de 3 anos mais o conteúdo esta top ainda parabéns!!! show montanooooo
